assets/js/img_clicker.js

```diff
@@ -0,0 +1,23 @@
+$(function () {
+    "use strict";
+
+    $(".containerTeam .imgMembers").click(function () {
+        var $src = $(this).attr("src"); 
+        var $description = $(this).siblings("#desc").text();
+        var $name= $(this).siblings("h4").text();
+        var $job= $(this).siblings(".job").text();
+        document.getElementById("descPopUp").innerHTML=$description;
```

Comment:
This is marked as security vulnerability. It's not a big deal but it would be good to find an alternative.